shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1175,8 +1175,16 @@ - (void)keyboardWillChangeFrame:(NSNotification*)notification {
 }
 
 - (void)keyboardWillBeHidden:(NSNotification*)notification {
-  // When keyboard hide, the keyboardWillChangeFrame function will be called to update viewport
-  // metrics. So do not call [self updateViewportMetrics] here again.
+  if (self.targetViewInsetBottom != 0) {
```

Comment:
I think we should add the same "isLocal" check in keyboardWillChangeFrame here: 

```
  if (@available(iOS 9, *)) {
    // Ignore keyboard notifications related to other apps.
    id isLocal = info[UIKeyboardIsLocalUserInfoKey];
    if (isLocal && ![isLocal boolValue]) {
      return;
    }
  }
```

Reply:
done